shortcircuit-api/src/test/java/com/powsybl/shortcircuit/json/JsonFaultParametersTest.java

```diff
@@ -44,7 +45,7 @@ public void readUnexpectedField() throws IOException {
         Files.copy(getClass().getResourceAsStream("/FaultParametersFileInvalid.json"), fileSystem.getPath("/FaultParametersFileInvalid.json"));
 
         Path path = fileSystem.getPath("/FaultParametersFileInvalid.json");
-        AssertionError e = assertThrows(AssertionError.class, () -> FaultParameters.read(path));
-        assertEquals("Unexpected field: unexpected", e.getMessage());
+        UncheckedIOException e = assertThrows(UncheckedIOException.class, () -> FaultParameters.read(path));
```

Comment:
@flo-dup, somehow here the transmission of the exception is different and the message is less clear. Do you know if there is a better way to do it? (it is the same problem with every exception message that I had to change)

Reply:
I don't know if the message is less clear? but if you want to get rid of his, you could disable the exception wrapping by calling, when creating your objectMapper:
```java
objectMapper.disable(DeserializationFeature.WRAP_EXCEPTIONS)
```

Reply:
I'm not sure I agree with this modification (which is not even useful now since the tests do not fail anymore with the passage to Junit5). The wrapped exception brings other information about the location of issues, are we sure we want to get rid of it?

Reply:
@colinepiloquet I agree with @miovd, id adds more information about the exception, so we should probably keep it. Even if in your case it's not that useful, for all other exceptions which occur there you won't have the full context.